Cargo.toml

```diff
@@ -19,3 +19,7 @@ url = "1.2"
 
 [dev-dependencies]
 env_logger = "0.3"
+
+[[example]]
+name = "simple"
+path = "examples/simple.rs"
```

Comment:
Both of these properties are the default, they are implied if you run `cargo run --example simple` without these values in this file.